docs/language_specification/about_the_grammar_sections.md

```diff
@@ -0,0 +1,14 @@
+1. In the syntax notation used in this documentation, syntactic categories are indicated by _italic_ type,
+and literal words and characters in <code>__boldface constant width__</code> type.</br></br>
+Alternatives are listed on separate lines except in a few cases
+where a long set of alternatives is presented on one line, with the help of "one of", "through" and "or". For example:</br>
+&nbsp;&nbsp;&nbsp;&nbsp;one of <code>__a__</code> through  <code>__z__</code> or <code>__A__</code> through  <code>__Z__</code></br>
+indicates any lowercase or uppercase alphabetic character.</br></br>
+An optional terminal or non-terminal symbol is indicated by the subscript "<sub>opt</sub>", so</br>
+&nbsp;&nbsp;&nbsp;&nbsp;<code>__{__</code> _expression<sub>opt</sub>_ <code>__}__</code></br>
+indicates an optional expression enclosed in braces.
+
+2. Names for syntactic categories have generally been chosen according to the following rules:
+     - _X-sequence_ is one or more _X_’s without intervening delimiters (e.g. _digit-sequence_ is a sequence of digits).
+     - _X-list_ is one or more _X_’s separated by intervening commas (e.g. _index-list_ is a sequence of
+indices separated by commas).
```

Comment:
```suggestion
     - _X-list_ is one or more _X_’s separated by intervening commas (e.g. _index-list_ is a sequence of comma separated indices).
```

Reply:
Nah, I don't like this change, sorry! :)

Reply:
Fair enough